cmd/xeniad/routes/routes.go

```diff
@@ -127,6 +123,8 @@ func routes(a *app.App) {
 
 	a.Handle("POST", "/v1/exec", handlers.Exec.Custom)
 	a.Handle("GET", "/v1/exec/:name", handlers.Exec.Name)
+	graphGroup.Handle("GET", "/v1/exec/:name/view/:view/:item", handlers.Exec.NameOnView)
+	graphGroup.Handle("POST", "/v1/exec/view/:view/:item", handlers.Exec.CustomOnView)
```

Comment:
I would suggest grouping the graphGroup stuff together if you can. If it truly is only used twice, it would make sense enough to just include cayley as route middleware instead.